### ISSUE TITLE
archlinux package outdated

### ISSUE THREAD

**Issue by loynoir** [koalaman/shellcheck#2210]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2155
- My shellcheck version (`shellcheck --version` or "online"): 0.7.1-360
- OS: archlinux
- package name: shellcheck

#### Reproduce Environment
To avoid configuration problem, reproduce in docker.
```sh
[host] # docker run -it archlinux
```
```sh
[container1] # pacman --noconfirm -Sy && pacman --noconfirm -S shellcheck
[container1] # pacman -Ss shellcheck
community/shellcheck 0.7.1-360 [installed]
    Shell script analysis tool
[container1] # echo -e '#!/bin/bash\nreadonly foo="$(mycmd)"\necho "${foo}"' > x.sh
[container1] # shellcheck x.sh  && echo 'BOOM! Expect an error!'
BOOM! Expect an error!
```

#### Here's a snippet or screenshot that shows the problem:
Problematic code extracted from [shellcheck wiki](https://github.com/koalaman/shellcheck/wiki/SC2155#problematic-code-in-the-case-of-readonly)
```sh
#!/bin/bash
readonly foo="$(mycmd)"
echo "${foo}"
```

#### Here's what shellcheck currently says:
Nothing, shellcheck is not complaining.

#### Here's what I wanted or expected to see:
```txt
In foobar.sh line 2:
readonly foo="$(mycmd)"
         ^-^ SC2155: Declare and assign separately to avoid masking return values.
```
### Links
[Packaging Source Code](https://github.com/archlinux/svntogit-community/blob/packages/shellcheck/trunk/PKGBUILD)

**Comment by loynoir** [comment]
@felixonmars

**Comment by TinCanTech** [comment]
> community/shellcheck 0.7.1-360 [installed]

```
 # shellcheck-v0.7.2/shellcheck -V
ShellCheck - shell script analysis tool
version: 0.7.2
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

**Comment by loynoir** [comment]
@felixonmars seems 0.7.1 is buggy, can you update PKGBUILD please?
```txt
$ ./shellcheck-v0.7.1/shellcheck x.sh && echo BOOM
BOOM
$ ./shellcheck-v0.7.2/shellcheck x.sh                                 

In x.sh line 2:
readonly foo="$(mycmd)"
         ^-^ SC2155: Declare and assign separately to avoid masking return values.
```

**Comment by loynoir** [comment]
In Apr 5, 2020, shellcheck v0.7.1 [released](https://github.com/koalaman/shellcheck/releases/tag/v0.7.1)
In Oct 24, 2020, support for `SC2155 readonly`  [added](https://github.com/koalaman/shellcheck/commit/619662adb63e027866a7511e695b87d8ca069654)
In 5 days ago, archlinux [packaging](https://github.com/archlinux/svntogit-community/blob/packages/shellcheck/trunk/PKGBUILD) using shellcheck v0.7.1 source code, that version has **no** support for `SC2155 readonly`.
In today, shellcheck v0.7.2 [released](https://github.com/koalaman/shellcheck/releases/tag/v0.7.2) which support `SC2155 readonly`

**Comment by koalaman** [comment]
v0.7.1 is a specific release with a specific feature set. It's not expected that a v0.7.1 package built later contains additional features. When a distro releases a "new" revision of 0.7.1, it's most likely the same version just rebuilt with updated library dependencies. 

If you are wondering when an official 0.7.2 package will be released for your distro, this is up to various volunteers working on the distro (thanks everyone!). Since 0.7.2 was released to Hackage today, give it at least a few weeks.

**Comment by loynoir** [comment]
You guys are really awesome. 😘
Sincerely apology.

**Comment by felixonmars** [comment]
@loynoir Just FYI that shellcheck 0.7.2 has been updated in Arch :)
